wire/carrier.go

```diff
@@ -0,0 +1,38 @@
+package wire
+
+// ProtobufCarrier is a DelegatingCarrier that uses protocol buffers as the
+// the underlying datastructur. The reason for implementing DelagatingCarrier
+// is to allow for users to serialize the underlying protocol buffers using
+// jsonpb or any other serialization forms they want.
+type ProtobufCarrier struct {
```

Comment:
It's not, I figured I'd go ahead and include it though since we're already providing the generated protobuf code.